Drive/wheel.hpp

```diff
@@ -43,17 +43,20 @@ class Wheel
 
   /// Sets the speed of the hub motor. Will not surpass max/min value
   /// @param hub_speed the new speed of the wheel
-  void SetHubSpeed(units::angular_velocity::revolutions_per_minute_t hub_speed)
+  void SetHubSpeed(int hub_speed)
   {
-    hub_speed_ = hub_speed;
+    units::angular_velocity::revolutions_per_minute_t hub_spd_to_rpm(hub_speed);
+    hub_speed_ = hub_spd_to_rpm;
     hub_motor_.SetSpeed(hub_speed_);
   }
 
   /// Adjusts the steer motor by the provided rotation angle/degree.
   /// @param rotation_angle positive angle (turn right), negative angle (left)
-  void SetSteeringAngle(units::angle::degree_t rotation_angle)
+  void SetSteeringAngle(int rotation_angle)
   {
-    steer_angle_ = rotation_angle + homing_offset_angle_;
+    // units::angle::degree_t steer_angle(rotation_angle + homing_offset_angle_);
+    units::angle::degree_t angle_into_deg(rotation_angle);
+    steer_angle_ = angle_into_deg;
```

Comment:
@Coreyboy1820 example of passing in arguments as int and then doing the conversion within this function to simplify rover_drive_system.cpp file